daml-script/daml3/BUILD.bazel

```diff
@@ -45,6 +45,17 @@ EOF
     for lf_version in COMPILER_LF2_VERSIONS
 ]
 
+# For convenience, we define daml3-script.dar which is always compiled to the
+# default stable version of LF2.
+genrule(
+    name = "daml3-script",
+    # TODO(#17366): use lf_version_default or equivalent once available
+    srcs = [":daml3-script-{}.dar".format(lf_version_latest.get("2"))],
+    outs = ["daml3-script.dar"],
+    cmd = "cp -L $(location :daml3-script-{}.dar) $$PWD/$@".format(lf_version_latest.get("2")),
```

Comment:
I was going to suggest an `alias` but then I realized that wouldn't change the name of the dar. Oh well, minor bazel annoyance.